packages/lodestar/src/sync/sync.ts

```diff
@@ -155,6 +154,7 @@ export class BeaconSync implements IBeaconSync {
 
   private syncCompleted = async (): Promise<void> => {
     this.stopSyncTimer();
+    this.chain.emitter.on("unknownBlockRoot", this.onUnknownBlockRoot);
```

Comment:
so this becomes an event for afterSync. Do we want to rename it to "afterSync:unknownBlockRoot" or "synced:unknownBlockRoot"?